### PR TITLE
[IMP]-15.0-Improve error message when update record in batch

### PR DESCRIPTION
**Pull request by hieulucky111**
Add one more option in the Mass Editing Wizard to allow user to be easy recognize which record is making error.
Ex:
When we update Product Template in Batch without checkbox "Write Record By Record"
image

When use check the checkbox "Write Record By Record"
image

### CODE REVIEW

**Comment by Leonidas-VII** [comment]
I like your idea, I can recognize which record is making errors easier. But it seems we need more automatic tests to pass the pipeline, no?

**Comment by Leonidas-VII** [comment]
> I like your idea, I can recognize which record is making errors easier. But it seems we need more automatic tests to pass the pipeline, no?

All tests are passed. Please ignore my comment above

**Review by Leonidas-VII — APPROVED** [review]
LGTM!

File: mass_editing/wizard/mass_editing_wizard.py [hunk 1]
```diff
@@ -17,6 +26,12 @@ class MassEditingWizard(models.TransientModel):
     operation_description_warning = fields.Text(readonly=True)
     operation_description_danger = fields.Text(readonly=True)
     message = fields.Text(readonly=True)
+    write_record_by_record = fields.Boolean(
+        help="This option will write the records one by one, instead of all at once.\n"
+        "This is useful when you are editing a lot of records and one of the "
+        "record raises an error. \n  With this option, the error message will be "
```
Comment (jcdrubay) [inline mass_editing/wizard/mass_editing_wizard.py]:
```suggestion
        "records raises an error. \n  With this option, the error message will be "
```

**Review by jcdrubay — APPROVED** [review]

**Review by quannguyen1560 — APPROVED** [review]

**Comment by pedrobaeza** [comment]
/ocabot merge minor

Please fw-port it to 16

**Pull request merged**